### PR TITLE
Fiscal sponsorship template letter

### PR DESCRIPTION
**Pull request by tmb**
This PR adds an auto-generated fiscal sponsorship letter to the Documents section of every event based on [this](https://docs.google.com/document/d/109mOH4f771mqu-UsvE7pXPEV07NA05ab9TV2OLEAXIA/edit) template.

out

cc @mdestefanis11

### CODE REVIEW

**Comment by maxwofford** [comment]
Just had a code review w/ @tmb & he has a list fixes to make before merging

**Comment by mdestefanis11** [comment]
Great! Thank you both for your hard work on Bank over the last few days.
Max, would love an update after code review 2 electric boogaloo.
--
Michael Destefanis
Hack Club Bank Director
https://hackclub.com

**Comment by lachlanjc** [comment]
@tmb Is this ready for my attention? Can you put your list of fixes here? I don’t want to spend time making design changes to code that’ll be cut/change.

**Comment by tmb** [comment]
@lachlanjc fixes are all permissions (policy) related, so it’s good for design changes!

**Comment by lachlanjc** [comment]
Made some design improvements:

fiscal_sponsorship_letter

**Comment by zachlatta** [comment]
Looking amazing. If we can, let’s have Michael’s signature on the letter so they see him as their POC.

Font looks broken I’m generated letter btw. Pretty sure we use sans serif?

**Comment by lachlanjc** [comment]
@zachlatta Yeah, sounds good.

I switched it to a serif (Spectral) to look more official—should I switch back to a sans-serif?

**Comment by tmb** [comment]
all backend changes are finished. @MaxWofford this is ready for review pending any design changes from @lachlanjc

**Comment by tmb** [comment]
update: just sent @lachlanjc the template tonight. aiming for a tomorrow afternoon EST QA (cc @MaxWofford )

**Comment by maxwofford** [comment]
Send me a cal invite for when you want to do the QA– include the list of bugs/issues Zach brought up in the description of the event

File: app/controllers/documents_controller.rb [hunk 1]
```diff
@@ -65,8 +65,9 @@ def download
     ).save!
   end
 
-  def fiscal_sponsorship_letter 
+  def fiscal_sponsorship_letter
     @documents = @event.documents.includes(:user)
+    puts @documents
```
Comment (lachlanjc) [inline app/controllers/documents_controller.rb]: Not sure if you meant to commit this

Comment (tmb) [reply app/controllers/documents_controller.rb]: oops :(

will remove

Comment (lachlanjc) [reply app/controllers/documents_controller.rb]:
```suggestion
```

**Comment by lachlanjc** [comment]
@tmb Fixed up the layout & switched back to sans-serif & removed Skeleton. Let me know if other changes are needed.

**Comment by zachlatta** [comment]
❤️

--
Zach Latta

https://zachlatta.com ( http://zachlatta.com )
@zachlatta ( https://twitter.com/zachlatta ) / fb ( https://www.facebook.com/crynix ) / github ( https://github.com/zachlatta )

**Comment by tmb** [comment]
[fiscal_sponsorship_letter.pdf](https://github.com/hackclub/bank/files/3193520/fiscal_sponsorship_letter.pdf)
Here's where we're at now. Pretty sure that we're done except for the small bit of text being lower than the logo (cc @lachlanjc). Once that's done it's QA + ship!

**Comment by tmb** [comment]
Code & QA mango approved by @MaxWofford (pending Zach)

**Comment by lachlanjc** [comment]
🥭

**Comment by tmb** [comment]
🚢 time

**Pull request merged**